cardano-cli/test/cardano-cli-golden/files/golden/help/debug_log-epoch-state.cli

```diff
@@ -0,0 +1,16 @@
+Usage: cardano-cli debug log-epoch-state --socket-path SOCKET_PATH
+                                           --node-configuration-file FILE
+                                           --out-file FILE
+
+  Log epoch state.
+
+Available options:
+  --socket-path SOCKET_PATH
+                           Path to the node socket. This overrides the
+                           CARDANO_NODE_SOCKET_PATH environment variable. The
+                           argument is optional if CARDANO_NODE_SOCKET_PATH is
+                           defined and mandatory otherwise.
+  --node-configuration-file FILE
+                           Input filepath of the node configuration file.
+  --out-file FILE          Output filepath of the log file.
```

Comment:
Can you add a mention that output is in JSON?

Reply:
Done!